### ISSUE TITLE
Firebase Error on LineageOS

### ISSUE THREAD

**Issue by damnms** [element-hq/element-x-android#4079]
### Steps to reproduce

1. Install Element X on a LineageOS mobile phone
2. Start it and login to your homeserver
3. After logging in, an error appears that push notifications might not be available because of "other error"
4. Click on "Settings" and click on "Fix" (i translated them myself from german -> english, so maybe they are not 100% the same wordings)
5. Error will appear

#### What did you expect?
No error at all

#### What happened instead?
An error and probably no push notifications at all

### Your phone model

OnePlus 11

### Operating system version

LineageOS 21

### Application version and app store

0.7.5

### Homeserver

Synapse 1.121.1 on debian trixie

### Will you send logs?

Yes

### Are you willing to provide a PR?

No

**Comment by spaetz** [comment]
Yes, so what is your problem? If you have LineageOS without google apps installed, there will be no Google push available. If you do have the whole Google shebang installed, it should be working.

**Comment by damnms** [comment]
It says firebase is available and a line below it says firebase not available. My assumption is that the text is misleading, if thats the correct behaviour.
I am not familiar with the push magic behind all those apps, so i assume firebase is the google push server. I installed ntfy as alternative, because thats the non-google push service, right? when firebase is not available, i would expect that to be used. Without an error.

**Comment by bmarty** [comment]
In the troubleshoot notification test suite, this test:

<img width="400" alt="image" src="https://github.com/user-attachments/assets/831abfd6-a76b-4ab0-b850-5fa80740ce6d" />

(in English)

<img width="400" alt="image" src="https://github.com/user-attachments/assets/899934a4-9a60-42df-92b7-20244eb3bac8" />

is actually confirming that the application is able to use Firebase, if it is installed on the device. It is a check that our internal module interacting with Firebase is included in the build.

I also think that the wording may be a bit confusing.

We may want to change the wording `Detect push providers` to something else like `Detect push provider plugins` or something else.

**Comment by damnms** [comment]
Thanks, that makes a bit more sense then. 
If my text is correct from https://github.com/element-hq/element-x-android/issues/4079#issuecomment-2558454340, it is still somehow confusing because i use LineageOS without any google services. Why is firebase available at all when its "google-free"? Is this a problem in LineageOS? The other thing is, firebase is available but seems like non-functioning? Why is UnifiedPush then not used? And why cant i change that in element x on android?

**Comment by bmarty** [comment]
Only the code in Element X Android is available to manage Firebase Push for you. But Firebase is not available, conformed by this other test:

<img width="817" alt="image" src="https://github.com/user-attachments/assets/4b8cea61-9b8b-42a7-9655-8d76171fb31e" />

For instance, if a user installs Element X Android from F-Droid (https://f-droid.org/packages/io.element.android.x/), the application will not have the code to manage Firebase, even if Firebase (Google Play Service) is available on their device.

**Comment by damnms** [comment]
I see, so firebase is available when i install element x from google play store but when i use fdroid, that function is not available at all? so why list it then there? for me, as a bit more technical user, i am highly confused by that and it looks like something is wrong with my mobile phone. Also, i would not say "Firebase ist nicht verfügbar" which means its not there at all, i would use something like "Firebase inkompatibel mit Custom-ROM" or so. 
I installed it from fdroid on a google-free phone, why is there firebase then and those checks? i am now really confused.
I will install it with aurora (google play store wrapper) and see how that behaves.

**Comment by bmarty** [comment]
Would it bit clearer with this wording?

<img width="395" alt="image" src="https://github.com/user-attachments/assets/c7913c97-7901-46d7-a065-498b5b44dc4e" />

**Comment by damnms** [comment]
Absolutely! Just a bit much text but it makes now 100% more sense (imo)

**Comment by bmarty** [comment]
`support` is the word I was looking for, thanks!
